### ISSUE TITLE
How or where to TRANSLATE to other languages?

### ISSUE THREAD

**Issue by heidiwenger** [Starz0r/AreWeAntiCheatYet#784]
About 95% of the world don't speak English as a first language, and 75% don't speak English at all, so these things, i think, are important.

Many projects use Weblate (which can be self hosted), Crowdin, Transifex or Launchpad. These are the most simplest ways to start translating easily and quickly for everyone. Thanks.

**Comment by Curve** [comment]
We don't have any internationalization support yet, however when I find the time to do so, I can setup weblate

**Comment by heidiwenger** [comment]
> We don't have any internationalization support yet, however when I find the time to do so, I can setup weblate

Great! Looking forward on spreading word that the possibility to translate has come. It's not that massive [web page](https://areweanticheatyet.com/), so the results would possibly benefit quickly.

**Comment by Curve** [comment]
> > We don't have any internationalization support yet, however when I find the time to do so, I can setup weblate
> 
> Great! Looking forward on spreading word that the possibility to translate has come. It's not that massive [web page](https://areweanticheatyet.com/), so the results would possibly benefit quickly.

Yeah, we have very few things to translate actually, apart from the status descriptions it basically just the status names

**Comment by heidiwenger** [comment]
> > > We don't have any internationalization support yet, however when I find the time to do so, I can setup weblate
> > 
> > 
> > Great! Looking forward on spreading word that the possibility to translate has come. It's not that massive [web page](https://areweanticheatyet.com/), so the results would possibly benefit quickly.
> 
> Yeah, we have very few things to translate actually, apart from the status descriptions it basically just the status names

Any estimate when you could set up a Weblate? For new comers these language things are seen also welcoming and, as stated, a massive amount of the world don't understand English though its everywhere.

**Comment by Curve** [comment]
> > > > We don't have any internationalization support yet, however when I find the time to do so, I can setup weblate
> > > 
> > > 
> > > Great! Looking forward on spreading word that the possibility to translate has come. It's not that massive [web page](https://areweanticheatyet.com/), so the results would possibly benefit quickly.
> > 
> > Yeah, we have very few things to translate actually, apart from the status descriptions it basically just the status names
> 
> Any estimate when you could set up a Weblate? For new comers these language things are seen also welcoming and, as stated, a massive amount of the world don't understand English though its everywhere.

No sorry, I'm very busy lately

I don't think it's too bad currently because basically the only thing people have to understand is the statuses (also, colors and icons help a lot here too in case one doesn't understand instantly)

**Comment by Domefemia** [comment]
How did it ended up? Has the weblate been setup?

**Comment by Curve** [comment]
> How did it ended up? Has the weblate been setup?

No, I did not have time to do so yet.
If I eventually get to it this issue will be used to track the progress.

**Comment by Curve** [comment]
I'm currently partially re-writing the website and with the new version it should be possible to easily adapt a nested dynamic route scheme where we can statically generate pages that use translated strings (I have not properly looked into this yet though, I will first finish the ground-work so that the page is ready to be used, and then look into this)

**Comment by heidiwenger** [comment]
Yes wonderful, finally =)
